internal/test/test.go

```diff
@@ -24,6 +24,31 @@ func CmpIntSlice(a, b []int) bool {
 	return true
 }
 
+// CmpIntSet returns true, if both sets have the smae entries
+func CmpIntSet(a, b map[int]bool) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	if len(a) == 0 && len(b) == 0 {
+		// handle nil case
+		return true
+	}
+
+	for key := range a {
+		if !b[key] {
+			return false
+		}
+	}
+
+	for key := range b {
```

Comment:
Line 43 to 47 can be removed. You already checked, that the maps have the same length and also, that every entry in map a is also in map b.